practice-one/src/components/menu.html

```diff
@@ -0,0 +1,18 @@
+<section class="menu-container">
+  <h2 class="menu-heading">
+    <span class="primary-color">Menu</span>
+     That 
+    <span class="accent-color">Always</span>
+     Make You Fall In 
+    <span class="primary-color">Love</span></h2>
```

Comment:
close tag `h2` should be on another line

Reply:
thanks for your comment